config/pmd-test.xml

```diff
@@ -195,15 +195,16 @@
 
     <rule ref="category/java/design.xml/UseObjectForClearerAPI">
         <properties>
-            <!-- This is checking for amount of arguments more that 3 (hardcoded).
-                    But in make sense to start making violations on items where
-                    amount of arguments more that 7. Extra abstraction(object)
-                    for single method does not make code more readable or
-                    easy to maintain -->
+            <!-- This is checking for amount of String arguments more than 3 (hardcoded).
+                 Extra abstraction(object) for single method does not make code more readable or
+                 easy to maintain. The rule should also only check public methods but has a bug.
+                 Suppress the false-positives. -->
```

Comment:
lets remove "The rule should also only check public methods but has a bug. Suppress the false-positives".
We just does not agree with this rule, if suppressions amount be alot, we will disable it..

Reply:
Well, it seems they changed mind since then.
I PRed a fix for this and it was accepted and merged.
Next version of PMD will have the bug fixed and actually the rule gets deprecated and in 7.x removed.
See https://github.com/pmd/pmd/issues/1760 and https://github.com/pmd/pmd/pull/1752.

Should I still remove this sentence from the comment as it is actually correct or leave it there?

Reply:
I am fine with keeping the sentence as it is then.